src/main/java/com/resturant/menu/controllers/OrderItemController.java

```diff
@@ -17,29 +17,29 @@ public OrderItemController(OrderItemService orderItemService){
         this.orderItemService = orderItemService;
     }
 
-    @RequestMapping(method= RequestMethod.GET, value="/orders")
+    @RequestMapping(method= RequestMethod.GET, value="/orderItems")
```

Comment:
technically we could nix this endpoint, but figured we can keep for now, just renaming routes